src/content/en/fundamentals/design-and-ui/input/touch/index.md

```diff
@@ -247,79 +138,66 @@ If, however, you expect users to interact with multiple elements at the same
+other elements have had the opportunity to catch and interpret the
+events (`true` means we want the event before other elements).
 
-Include these event names in the `addEventListener()` method, along with the
-event’s callback function and a boolean. The boolean determines whether you
-should catch the event before or after other elements have had the
-opportunity to catch and interpret the events (`true` means we want the event
-before other elements).
+Here's an example of listening for the start of an interaction.
 
 <pre class="prettyprint">
 {% includecode content_path="web/fundamentals/design-and-ui/input/touch/_code/touch-demo-1.html" region_tag="addlisteners" adjust_indentation="auto" %}
 </pre>
 
-This code first checks to see if Pointer Events are supported by testing for
-`window.PointerEventsSupport`, if Pointer Events aren’t supported, we add listeners for
-touch and mouse events instead.
+Note: PointerEvents only need a single pointerdown event to handle both mouse and touch events. This is due to the design of the API.
 
-The value `window.PointerEventSupport` is determined by looking for the
-existence of `window.PointerEvent` or the now deprecated
-`window.navigator.msPointerEnabled` objects. If they are supported we use
-varibles for event names, which use the prefixed or unprefixed versions depending
-on the existence of `window.PointerEvent`.
+#### Handle Single-Element Interaction
 
-<pre class="prettyprint">
-{% includecode content_path="web/fundamentals/design-and-ui/input/touch/_code/touch-demo-1.html" region_tag="pointereventsupport" adjust_indentation="auto" %}
-</pre>
+In the short snippet of code above we only added the starting event listener,
+this wasn't just to show you how to add the event listeners. for single
+element gestures, this is the only event listener we add at first.
 
-### Handle Single-Element Interaction
-
-In the short snippet of code above you may have noticed that we only add the starting event listener, this is a conscious decision.
-
-By adding the move and end event listeners once the gesture has
-started on the element itself, the browser can check if the touch occured in a
-region with a touch event listener and if it’s not, can handle it faster
-by not having to run any additional javascript.
+By adding the "move" and "end" event listeners *after* a gesture has
+started, the browser can quickly check if the touch occurred in a
+region with an event listener and if not, it can skips calling
+any JavaScript, which can be a slow process.
 
 The steps taken to implement this are:
 
 1. Add the start events listener to an element.
-1. Inside your touch start method, bind the move and end elements to the
+1. Inside your start gesture callback, bind the move and end elements to the
    document. The reason for binding the move and end events to the
    document is so that we receive all events regardless of whether they
    occur on the original element or not.
```

Comment:
Yeah I think this hints at the real reason to generally prefer adding move/end listeners from within the start callback.  Though it's different depending on the type of event (probably more detail than you want to go into here):
- For mouse events, this is the only way you'll get events if the pointer gets ahead of the element you're dragging (as you've described)
- For pointer events you're probably better off using 'setPointerCapture' to indicate you want the target to capture all events. 
- For touch, we do the setPointerCapture implicitly (both for touch events and pointer events, with Edge changing to match us in their pointer events implementation).

So maybe to make this more accurate, just clarify "so that we receive all _mouse_ events regardless ...".